plugin/src/sbt-test/sbt-jni/simple-cargo/build.sbt

```diff
@@ -0,0 +1,11 @@
+ivyLoggingLevel := UpdateLogging.Quiet
+
+lazy val root = (project in file(".")).aggregate(core, native)
+
+lazy val core = project
+  .settings(libraryDependencies += "org.scalatest" %% "scalatest" % "3.2.9" % Test)
+  .dependsOn(native % Runtime)
+
+lazy val native = project
+  .settings(nativeCompile / sourceDirectory := baseDirectory.value) // `baseDirectory`, not `sourceDirectory`
```

Comment:
Ha, what a trick, indeed, that's due to the Cargo project structure (mostly a comment for myself):
```
├── Cargo.lock
├── Cargo.toml
├── src/
│   ├── lib.rs
```